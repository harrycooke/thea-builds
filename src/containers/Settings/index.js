import React from "react";

//why are these imports made here???

// import ConsultsCard from '../../components/Dashboard/Utilization/ConsultsCard';
// import TurnaroundCard from '../../components/Dashboard/Utilization/TurnaroundCard';
// import SavingsConsultsCard from '../../components/Dashboard/Savings/ConsultsCard';
// import SavingsCostsCard from '../../components/Dashboard/Savings/CostCard';
// import FollowupsCard from '../../components/Dashboard/Efficiency/FollowupsCard';
// import QualityCard from '../../components/Dashboard/Efficiency/QualityCard';
import {
  Grid,
  Typography
} from '@material-ui/core';
import { withStyles } from "@material-ui/core/styles";
import styles from "../../assets/styles/DashboardStyles";

class Settings extends React.Component {

  state = {
    open: false
  };

  handleToggleDrawer = () => {
    this.setState(prevState => {
      return { open: !prevState.open };
    });
  };

  render() {

    const { classes } = this.props;

    return (
      <div className={classes.root} >
        <img src={require("../../assets/images/dashboard.png")} alt="required" style={{width: "100%"}} />
      </div>
    );
  }
};

export default withStyles(styles)(Settings);
