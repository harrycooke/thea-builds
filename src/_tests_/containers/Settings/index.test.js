import React from 'react';
import ReactDOM from'react-dom';
import TestRenderer from 'react-test-renderer'; // ES6
import {Settings} from '../../../containers/Settings/index';

describe('Settings', () => {
    beforeAll = () => {
        state = {
            open: false
          };
    };


    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(Settings, div);
    });

    it('renders React components, traversing a ReactElement tree and represents as JSON object', () => {
        const renderer = TestRenderer.create(Settings);
        renderer.toJSON();
      });
      
    describe('Settings', () => {
        it('should render correctly in "debug" mode', () => {
          const component = (Settings);
          expect(component).toMatchSnapshot();
        });
      });
});