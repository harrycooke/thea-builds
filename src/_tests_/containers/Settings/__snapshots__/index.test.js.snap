// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings Settings should render correctly in "debug" mode 1`] = `undefined`;
